applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -133,7 +133,12 @@ Response syntax
 
 ::
 
-   #XMQTTCON: <client_id>,<url>,<port>[,<sec_tag>]]
+   #XMQTTCON: <status>,<client_id>,<url>,<port>[,<sec_tag>],<keepalive>]
+
+* The ``<status>`` value is an integer.
+
+    * ``0`` - MQTT disconnected.
+    * ``1`` - MQTT connected.
```

Comment:
```suggestion
    * ``0`` - MQTT is not connected.
    * ``1`` - MQTT is connected.
```
This presumes that the read command is actively used to gain this information. If this is instead something that gets automatically reported when MQTT connects or disconnects, then the original, active wording is correct.

Reply:
The read command is just to report, not get input

Reply:
What I was more looking for, is this command used by the user? Or is it something that gets automatically run?

Reply:
Yes it's for user to call
AT#XMQTTCON=....   SET comannd
AT#XMQTTCON?       READ command
AT#XMQTTCON=?    TEST command